src/index/ivf_raft/ivf_raft.cuh

```diff
@@ -540,5 +552,11 @@ class RaftIvfIndexNode : public IndexNode {
     std::optional<T> gpu_index_;
 };
 
+void
+SetRaftMemPool(size_t init_size, size_t max_size) {
```

Comment:
Who's the caller?

Reply:
call from knowhere_config, milvus will call it.